op/model.py

```diff
@@ -307,57 +314,39 @@ def __init__(self):
         self._is_leader = None
         self._leader_check_time = 0
 
-    def _run(self, *args, capture_output=True, use_json=True):
-        if capture_output:
-            kwargs = dict(stdout=PIPE, stderr=PIPE)
-            if use_json:
-                args += ('--format=json',)
-        else:
-            kwargs = dict()
-        result = run(args, check=True, **kwargs)
-        if capture_output:
-            text = result.stdout.decode('utf8')
-            if use_json:
-                return json.loads(text)
+    def _run(self, *args, return_output=True, use_json=True):
+        kwargs = dict(stdout=PIPE, stderr=PIPE)
+
+        if use_json:
+            args += ('--format=json',)
+
+        try:
+            result = run(args, check=True, **kwargs)
+        except CalledProcessError as e:
+            raise ModelError.from_exception(e)
```

Comment:
May do exception chaining here `ModelError.from_exception(e) from e` to have the original cause of the error.